minetest/minetest
# Add debug.txt.1 to .gitignore

Now that #8904 is merged, a file named `debug.txt.1` can be created automatically. This file should be ignored by git.

## To do

This PR is a Ready for Review.

## How to test

- Fill your `debug.txt` file until the thing happens and try to make a commit with other changes.
- Before the PR: Be annoyed that the file is added.
After: The file should not be added.
